### PR TITLE
Vote for the logo of DP-GEN

### PR DESCRIPTION
**Pull request by likefallwind**
As software that generates a deep learning-based model of interatomic potential energy and force field, DP-GEN can automatically prepare scripts, maintain job queues on HPC machines (High-Performance Cluster), and analyze results. Version v0.11.0 of DP-GEN was also updated last month. We have designed a logo for DP-GEN. Here is the latest version of it. Please determine whether you agree to use it. 

DP-GEN logo

### CODE REVIEW

File: votes/DMFF logo_vote.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# A vote for community Logo: DMFF
```
Comment (njzjz) [inline votes/DMFF logo_vote.md:1]: Why is DMFF added here?

File: votes/DP-GEN logo_vote.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# A vote for community Logo: DP-GEN
+ 
+## Proposal
+As software that generates a deep learning-based model of interatomic potential energy and force field, DP-GEN can automatically prepare scripts, maintain job queues on HPC machines (High-Performance Cluster), and analyze results. Version v0.11.0 of DP-GEN was also updated last month. We have designed a logo for DP-GEN. Here is the latest version of it. Please determine whether you agree to use it. 
+
+![DP-GEN logo](https://dp-public.oss-cn-beijing.aliyuncs.com/community/logo/DP-GEN.png)
```
Comment (njzjz) [inline votes/DP-GEN logo_vote.md:6]: Is there an SVG format?

**Review by wanghan-iapcm — COMMENTED** [review]
The image is not accessible.

File: votes/DMFF logo_vote.md (hunk 1 above)
Comment (likefallwind) [reply votes/DMFF logo_vote.md]: Sorry, it will be added in another PR, deleted it now

File: votes/DP-GEN logo_vote.md (hunk 2 above)
Comment (likefallwind) [reply votes/DP-GEN logo_vote.md:6]: Yes, the address is added now.

**Comment by likefallwind** [comment]
> The image is not accessible.

It is accessible now.

**Comment by njzjz** [comment]
It should be noted that the texts in this SVG version have different fonts on different computers. The texts should be expanded, like the DeePMD-kit logo in #21.

```svg
<text class="cls-4" transform="translate(465.82 262.2) scale(1.05 1)">DP</text><g class="cls-5"><text class="cls-4" transform="translate(785.76 265.16) scale(1.05 1)">GEN</text>
```

Screenshot from 2023-01-11 15-39-43

See also https://graphicdesign.stackexchange.com/a/27239.

**Comment by wanghan-iapcm** [comment]
+1

**Comment by likefallwind** [comment]
> It should be noted that the texts in this SVG version have different fonts on different computers. The texts should be expanded to shapes, like the DeePMD-kit logo in #21. (Other logos do not have this issue)
> 
> ```
> <text class="cls-4" transform="translate(465.82 262.2) scale(1.05 1)">DP</text><g class="cls-5"><text class="cls-4" transform="translate(785.76 265.16) scale(1.05 1)">GEN</text>
> ```
> 
> Screenshot from 2023-01-11 15-39-43
> 
> See also https://graphicdesign.stackexchange.com/a/27239.

ok I will check it

**Comment by likefallwind** [comment]
> It should be noted that the texts in this SVG version have different fonts on different computers. The texts should be expanded to shapes, like the DeePMD-kit logo in #21. (Other logos do not have this issue)
> 
> ```
> <text class="cls-4" transform="translate(465.82 262.2) scale(1.05 1)">DP</text><g class="cls-5"><text class="cls-4" transform="translate(785.76 265.16) scale(1.05 1)">GEN</text>
> ```
> 
> Screenshot from 2023-01-11 15-39-43
> 
> See also https://graphicdesign.stackexchange.com/a/27239.

Done, have a look？

**Review by njzjz — APPROVED** [review]
LGTM

**Pull request merged**